src/components/modules/banners/digitalSubscriptions/DigitalSubscriptionsBanner.tsx

```diff
@@ -120,7 +134,7 @@ export const DigitalSubscriptionsBanner: React.FC<BannerProps> = ({
                         <div css={bottomRightComponent}>
                             <div css={packShot}>
                                 <img
-                                    src="https://i.guim.co.uk/img/media/773ead1bd414781052c0983858e6859993870dd3/34_72_1825_1084/1825.png?width=500&quality=85&s=24cb49b459c52c9d25868ca20979defb"
+                                    src="https://media.guim.co.uk/3e6ecc7e48f11c5476fc2d7fad4b3af2aaff4263/4001_0_1986_1193/500.png"
```

Comment:
We now have final images:
ROW: https://media.guim.co.uk/22841f3977aedb85be7b0cf442747b1da51f780f/0_0_2320_1890/500.png
Australia: https://media.guim.co.uk/a1679c54314643a7b611a0c97b53a8ea38b9ec23/0_0_2320_1890/500.png

You probably want to run these through the image resizer: http://image-url-signer.s3-website-eu-west-1.amazonaws.com/

Let me know if that doesn't make sense!

Reply:
Actually, it looks like the Aus image may not be final! I just saw an email from Sherri

Reply:
I don't think we'll need to worry about the Aus image here for the time being, as we're still only showing them the GW banner at launch time. Thanks for the final image for everywhere else!